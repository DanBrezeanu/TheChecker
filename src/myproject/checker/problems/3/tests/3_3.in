12
1920 4924 5639 3177 7235 138 6934 6924 2173 8603 2402 6968 
