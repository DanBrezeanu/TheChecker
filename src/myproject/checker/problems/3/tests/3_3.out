138 1920 2402 4924 6924 6934 6968 
8603 7235 5639 3177 2173 
